README.md

```diff
@@ -1,2 +1,34 @@
 # helper-bot
+05/19/2021
+
+
 Use Discord Slash Commands to offer people helpful advice
+
+To add your bot to your discord server with the appropriate permissions, ensure that the permission for `applications.commands` exists in addition to the `bot` OAUTH scope. Without this you will receive 
+
+    "403 Forbidden (error code: 50001): Missing Access"
+
+### Set-Up
+In order to run this bot you need `Python >= 3.0` and the required dependencies installed.
```

Comment:
```suggestion
In order to run this bot you need `Python >= 3.6` and the required dependencies installed.
```

F-Strings were introduced in Python 3.6 therefore any below would not work: https://docs.python.org/3/whatsnew/3.6.html